CSE-Hub/home/signup.php

```diff
@@ -0,0 +1,73 @@
+<?php 
+
+require_once '../sql_login/login.php'	;
+
+$conn = mysqli_connect($hostname,$username,$password,$database);
+
+if(!$conn) {
+    die("Error occured while registration, please try again. <br>".mysqli_connect_error());
+}
+
+
+if(isset($_POST['register_uname']) && isset($_POST['register_pass']) && isset($_POST['register_name']) && isset($_POST['register_email']))
+{
+	$username = $_POST['register_uname'];
+	$password = $_POST['register_pass'];
```

Comment:
change the variable names here for 'register_uname' and 'register_pass' , its using the username and password from login.php and storing them in the tables instead of 'register_uname' and 'register_pass'

Reply:
Strange! That did not happen on my system. Anyways, changing it.